src/test/java/org/broadinstitute/hellbender/utils/pileup/ReadPileupUnitTest.java

```diff
@@ -328,6 +329,13 @@ public void testSimplePileupWithIndelsOffset(){
         final int offset = 4;
         final ReadPileup pu = new ReadPileup(loc, reads, offset);
 
+        final PileupElement firstElem = pu.getElementForRead(read1);
+        Assert.assertNull(firstElem.getAdjacentOperator(PileupElement.Direction.NEXT));
+        Assert.assertNull(firstElem.getAdjacentOperator(PileupElement.Direction.PREV));
+        final PileupElement secondElem = pu.getElementForRead(read2);
+        Assert.assertEquals(secondElem.getAdjacentOperator(PileupElement.Direction.NEXT), CigarOperator.I);
+        Assert.assertNull(secondElem.getAdjacentOperator(PileupElement.Direction.PREV));
```

Comment:
Replace with high-quality direct testing on `getAdjacentOperator()` in `PileupElementUnitTest`